install/resources/upgrade50.php

```diff
@@ -22,14 +22,16 @@
 
 function upgrade50_dbchanges()
 {
-	global $output, $cache;
+	global $output, $cache, $db;
 
-	$output->print_header("Updating Cache");
+	$output->print_header("Updating Database");
 
 	echo "<p>Updating cache...</p>";
 
 	$cache->delete("banned");
 
+	$db->update_query('settings', array('value' => 1), "name='nocacheheaders'");
```

Comment:
I'm not sure we want to force this setting on if a user purposefully wants it disabled for some reason...

Reply:
Yeah, I do not why but there could be some users whose wanna do that...

Reply:
Note the default value has changed - it wouldn't revert explicit preferences.

It may be the safer option, compared to adding another bullet point to the [security checklist](https://docs.mybb.com/1.8/administration/security/protection/) for existing installations, even if we mentioned it in release notes.